src/enigma2/Recordings.h

```diff
@@ -38,42 +38,46 @@ namespace enigma2
   static int CUTS_LAST_PLAYED_TYPE = 3;
   static int E2_DEVICE_LAST_PLAYED_SYNC_INTERVAL_MIN = 300;
   static int E2_DEVICE_LAST_PLAYED_SYNC_INTERVAL_MAX = 600;
+  static std::string TRASH_FOLDER = ".Trash";
```

Comment:
constexpr static std::string

Reply:
This would have an error as it's not a string literal.

Reply:
What speaks against making it char*?

Reply:
You mean still using constexpr? Then you get this: `warning: ISO C++11 does not allow conversion from string literal to 'char *const' [-Wwritable-strings]`

Reply:
`constexpr static const char* TRASH_FOLDER = ".Trash";`

Reply:
This works: `constexpr static const char*`!

Reply:
Starting with 'static' is more common, but this is only sugar ofc. => static constexpr const char*